src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -115,11 +115,17 @@ public static double Abs(Complex value)
                 return double.PositiveInfinity;
             }
 
+            return Hypot(value._real, value._imaginary);
+        }
+
+        private static double Hypot (double a, double b)
```

Comment:
Agreed. (With space is my preferred personal coding style, which crept in here, but I agree that the coding style of the project should take precedence.) Will change if another PR update is required, but not otherwise.